models/public/license-plate-recognition-barrier-0007/license-plate-recognition-barrier-0007.md

```diff
@@ -208,6 +208,20 @@ is an integer number encoding a character according to this dictionary:
         68 Y
         69 Z
 
+## Download a Model and Convert it into OpenVINOâ„¢ Inference Engine Format
+
+You can download models and if necessary convert them into Inference Engine format using the OpenVINOâ„¢ [Model Downloader and other automation tools](../../../tools/downloader/README.md) as shown in the examples below.
```

Comment:
Corrupted characters here.

Reply:
can we use `&trade;` symbols everywhere instead of those, directly coded in some codepage?

Reply:
Let's remove `OpenVINO` from headings and Model Downloader name:

- Download a Model and Convert it into OpenVINOâ„¢ Inference Engine Format --> Download a Model and Convert it into Inference Engine Format 
- OpenVINO(tm) Model Downloader --> Model Downloader

Reply:
I agree with @andrew-zaytsev, but I'll answer the question anyway:

> can we use `&trade;` symbols everywhere instead of those, directly coded in some codepage?

I slightly prefer using the real characters, because it looks better in the text editor. Our CI ensures that all files are encoded with UTF-8, so you can't mess up the encoding - the only reason these characters are wrong is because you inserted the wrong characters to begin with.

I don't see any reason to mass-replace existing non-ASCII characters with character entity references (like `&trade;`), although I don't mind if individual authors use them.

Reply:
Agree with Roman, I also prefer using the real characters (in the openvino repo we do)

Reply:
applied suggested changes